receiver/sqlserverreceiver/scraper_test.go

```diff
@@ -438,3 +447,69 @@ func TestInvalidQueryTextAndPlanQuery(t *testing.T) {
 	_, err := scraper.ScrapeLogs(context.Background())
 	assert.Error(t, err)
 }
+
+func TestRecordDatabaseSampleQuery(t *testing.T) {
```

Comment:
`TestRecordDatabaseSampleQuery` and `TestRecordInvalidDatabaseSampleQuery` could probably be in the same test if you use a test struct, have the test name, file name, and anything else you need in it. The setup is pretty much the same from what I can tell.